src/libutil/typedesc.cpp

```diff
@@ -388,6 +403,68 @@ sprintt(TypeDesc type, const char* format, const tostring_formatting& fmt,
 
 
 
+template<class T>
+inline std::string
+formatt(TypeDesc type, const char* format, const tostring_formatting& fmt,
```

Comment:
Maybe call this `format_type`?

Reply:
I can, though it's a strictly internal utility function to that module, and was named that way to be symmetric to the old one called sprintt.

Reply:
Right, same thought for `sprintt` -- it just a bit hard to read visually I think with that double `t`. At first I thought it was some other variant of `format` rather than a local function. `sprint_type` is more verbose but more immediately clear what it does.